documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -5,10 +5,16 @@
 
 The SAMM CLI is a command line tool for the validation of Aspect Models and the generation of artifacts, such as documentation or code, from Aspect Models.
 
-TIP: Download latest version: icon:download[] https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}.jar[samm-cli-{esmf-sdk-version}.jar]
+The SAMM CLI is available as a native executable for multiple platforms and as an an executable jar:
+
```

Comment:
The `TIP:` and `NOTE:` are Asciidoc syntax to create these styled blocks with the icons:
![image](https://github.com/eclipse-esmf/esmf-sdk/assets/1199036/ae27b5b1-5e98-4eba-9dd0-40383e340cc4)

Using `WINDOWS:` and `LINUX:` like this will look strange. I propose we put all download links as a list in the TIP section, i.e.:

```asciidoc
[TIP] 
.Downloading samm-cli
====
* Download executable for Windows (latest): https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}-windows-x86_64.zip[samm-cli-{esmf-sdk-version}-windows-x86_64.zip]
* Download exectuable for Linux (latest): https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}-linux-x86_64.tar.gz[samm-cli-{esmf-sdk-version}-linux-x86_64.tar.gz]
* Download executable jar for other platforms and architectures (latest): https://github.com/eclipse-esmf/esmf-sdk/releases/download/v{esmf-sdk-version}/samm-cli-{esmf-sdk-version}.jar[samm-cli-{esmf-sdk-version}.jar]. Running this version requires Java 17.
====
```